src/logging/mod.rs

```diff
@@ -6,7 +6,7 @@
 //!
 //! A [`Logger`] is configured by the [`LogConfig`], which in turn consist of [`Condition`]s
 //! and [`EntryExtractor`]s.
-//! See their respective documentation pages for more information on how to use them.
+//! See their documentation  for more information on how to use them.
```

Comment:
There is an additional whitespace after documentation? Sorry for being pedantic.

Reply:
No problem, it does look irritating.